components/FavoritesList.js

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import MovieDetails from "./MovieDetail";
+
+const ListContainer = styled.div`
+  margin-top: 20px;
+`;
+
+const ListItem = styled.div`
+  display: flex;
+  align-items: center;
+  padding: 5px;
+  border: 1px solid #ccc;
+  border-radius: 5px;
+  margin-bottom: 5px;
+  cursor: pointer;
```

Comment:
I would put styles in a seperate styles.js. so your files arent that big in the end. but its your decision :)